tests/hook_main.py

```diff
@@ -1361,3 +1361,55 @@ def event_sub_topic_delete(transaction):
         event_sub_topic = EventSubTopicFactory()
         db.session.add(event_sub_topic)
         db.session.commit()
+
+# ------------------------- Ticket Fees -------------------------
+
+
+@hooks.before("Ticket Fees > Ticket Fees Collection > List Ticket Fees")
+def ticket_fees_get_list(transaction):
+    """
+    GET /ticket-fees
+    :param transaction:
+    :return:
+    """
+    transaction['skip'] = True
```

Comment:
Can you please implement the Dredd testing as well? Or do you plan to do that as a separate PR?